### PR TITLE
Fix broken links; Improve overview page

### PR DESCRIPTION
**Pull request by spaceisfun**

### CODE REVIEW

File: docs/docs/index.md [hunk 1]
```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
...
 
-Factory Orchestrator provides a simple and reliable way to run and manage factory line validation and fault analysis workflows. Beyond the factory floor, Factory Orchestrator can be used during OS and hardware development to support various developer inner-loop and diagnostics activities.
-
-Factory Orchestrator consists of two components:
 
-- A .NET Core system service (Microsoft.FactoryOrchestrator.Service.exe): The service tracks task information, including run unique per-run results and logging; even persisting task state to allow the service to be resilient to data loss due to client failure. It also provides a robust API surface for clients to monitor & interact with the service via C# .NET, C# UWP, or PowerShell code. 
+Factory Orchestrator provides a simple and reliable way to run and manage factory line validation and fault analysis workflows. Beyond the factory floor, Factory Orchestrator can be used during OS and hardware development to support various developer inner-loop and diagnostics activities.
```
Comment (hajya) [inline docs/docs/index.md:10]: slight re-write:

"Factory Orchestrator provides a simple and reliable way to run validation or calibration software on target devices on a manufacturing line. Factory Orchestrator makes auditing and fault analysis by providing durable and easily identifiable 'per run' results and logs. Beyond the factory floor, Factory Orchestrator can be used during device development to support various developer inner-loop and diagnostics activities enabling further reuse of collateral created and leveraged across the full device manufacturing lifecycle.' 

feel free to re adjust as well. :)

File: docs/docs/index.md [hunk 2]
```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
 ---
 
 # Factory Orchestrator
+Factory Orchestrator, is a cross-platform service for organizing, executing, and logging a set of executable scripts, binaries, or ["Tasks"](tasks-and-tasklists.md) on a system. Factory Orchestrator tracks task information, including run unique per-run results and logging; even persisting task state to allow the service to be resilient to data loss due to system failure. Factory Orchestrator also provides an optional [UWP client app](use-the-factory-orchestrator-app.md) and [robust client API surface](use-the-factory-orchestrator-api.md) for clients to monitor & interact with the service via the App or C# .NET, C# UWP, or PowerShell code. The UWP app and and any other client can communicate with any Factory Orchestrator service running on the same system and/or over a network!
```
Comment (hajya) [inline docs/docs/index.md:7]: >  is a cross-platform service for organizing, 

does service mean the same to everyone?

could we replace with 'toolset'? or 'tooling'?

File: docs/docs/index.md [hunk 3]
```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
 ---
 
 # Factory Orchestrator
```
Comment (hajya) [inline docs/docs/index.md:6]: Consider adding something like this at the very very start before 'Factory Orchesators, is a cross'...:

"Built to allow device manufactures and developers to focus more on their validation and calibration software, and less on how to run, audit and manage the lifecycle of the applications..."

trying to point out 'what its for' before 'what it does'.

File: docs/docs/index.md [hunk 4]
```diff
@@ -4,27 +4,12 @@ ms.date: 09/30/2019
...
-
-The Task log files contain details about the execution of a specific of the Factory Orchestrator Task. There is one log file generated for each run of a Task ([TaskRun](../CoreLibrary/CoreLibrary\Microsoft-FactoryOrchestrator-Core-TaskRun/)). The files are saved to `%ProgramData%\FactoryOrchestrator\Logs\` on a device by default, but this location can be changed using the [FactoryOrchestratorClient](../ClientLibrary/Microsoft-FactoryOrchestrator-Client-FactoryOrchestratorClient-FactoryOrchestratorClient%28System-Net-IPAddress_int%29/).[SetLogFolder](../ClientLibrary/Microsoft-FactoryOrchestrator-Client-FactoryOrchestratorClient-SetLogFolder%28string_bool%29/)() API. Use the [FactoryOrchestratorClient](../ClientLibrary/Microsoft-FactoryOrchestrator-Client-FactoryOrchestratorClient-FactoryOrchestratorClient%28System-Net-IPAddress_int%29/).[GetLogFolder](../ClientLibrary/Microsoft-FactoryOrchestrator-Client-FactoryOrchestratorClient-GetLogFolder%28%29/)() API to programmatically retrieve the log folder.
+See [Getting started with Factory Orchestrator](get-started-with-factory-orchestrator.md) for details on how to install and run the UWP app and/or service.
```
Comment (hajya) [inline docs/docs/index.md:15]: i think there is more harm than benefit of saying its a 'UWP' app.

i think its fair to just say 'install and run the client application and/or service'

**Review by hajya — COMMENTED** [review]
some suggestion on the overveiw but overall looks good. :)

File: docs/docs/index.md (hunk 1 above)
Comment (hajya) [reply docs/docs/index.md:10]: can also use 'device under test' as it seems like we could link to wikipedia/its known in public domain: https://en.wikipedia.org/wiki/Device_under_test

**Review by hajya — COMMENTED** [review]
the overview page updates could likely be reflected into readme.md as well?

File: docs/docs/index.md (hunk 2 above)
Comment (spaceisfun) [reply docs/docs/index.md:7]: changed to "system service" which based on my searching seems pretty prolific.

File: docs/docs/index.md (hunk 4 above)
Comment (spaceisfun) [reply docs/docs/index.md:15]: changed initial to "client app for Windows", rest just "app"

**Comment by spaceisfun** [comment]
> Tasks in a TaskList can be configured to run

@patelp3989 good find, thanks!

**Pull request merged**